schedule/yast/opensuse/yast2_gui/yast2_gui.yaml

```diff
@@ -15,20 +15,27 @@ schedule:
   - console/prepare_test_data
   - console/consoletest_setup
   - console/hostname
+  - console/install_packages_simple
+  - console/firewalld_add_port
   - console/setup_libyui_running_system
+  - yast2_gui/yast2_firewall_stop_service
```

Comment:
let's not change the order where we schedule firewall tests, we might affect some other tests.